lib/workers/repository/onboarding/pr/__snapshots__/config-description.spec.ts.snap

```diff
@@ -9,7 +9,7 @@ Based on the default config's presets, Renovate will:
   - Start dependency updates only once this onboarding PR is merged
   - Run Renovate on following schedule: before 5am
 
-🔡 Would you like to change the way Renovate is upgrading your dependencies? Simply edit the \`.github/renovate.json\` in this branch with your custom config and the list of Pull Requests in the "What to Expect" section below will be updated the next time Renovate runs.
+🔡 Would you like to change the way Renovate is upgrading your dependencies? Simply edit the \`.github/renovate.json\` in this branch with your custom config and select the Retry/Rebase checkbox below, the list of Pull Requests in the "What to Expect" section below will be updated the next time Renovate runs.
```

Comment:
How about something like this?

- Use present tense
- Write short sentences
- Use simple words
- Drop words like `simply` (if we need to explain, it's not that simple 😄)

> Do you want to change how Renovate upgrades your dependencies? Put your custom config in the \`.github/renovate.json\` file in this branch and select the Retry/Rebase checkbox below. Renovate will update the list of Pull Requests in the "What to Expect" section the next time it runs.

Reply:
Additionally, the list of pull requests is not the only thing that can changes. For eg, if you ignore a file, it will be removed from the detected list. We can maybe simplify it like

```
🔡 Do you want to change how Renovate upgrades your dependencies? Put your custom config in the `.github/renovate.json` file in this branch and select the Retry/Rebase checkbox below. Renovate will update the Pull Request description the next time it runs.
```

Reply:
```suggestion
🔡 Do you want to change how Renovate upgrades your dependencies? Put your custom config in the `.github/renovate.json` file in this branch and select the Retry/Rebase checkbox below. Renovate will update the Pull Request description the next time it runs.
```

Reply:
I like your proposal. I'll let a a maintainer review and apply the suggestion. 😉

Reply:
cc @rarkins @RahulGautamSingh

Reply:
I don't think we need to select the checkbox for the PR body to be updated. Can you create a minimal reproduction of this?